config.env.py

```diff
@@ -9,6 +9,7 @@
 DEBUG = False
 IP = environ.get("PACKET_IP", "localhost")
 PORT = environ.get("PACKET_PORT", "8000")
+PROTOCOL = environ.get("PACKET_PROTOCOL", "https://")
```

Comment:
Why does this need to be a thing? Couldn't we just always assume `https`? Not questioning it; just curious.

Reply:
When testing locally we need to use http, so it will almost always be https, but can't be guaranteed